pyfr/integrators/controllers.py

```diff
@@ -49,6 +52,14 @@ def __init__(self, *args, **kwargs):
         # Delete the memory-intensive elements map from the system
         del self.system.ele_map
 
+    def set_dt(self, dt):
```

Comment:
Misleading method name.  It does nothing close to setting `dt`.  Rather, it appends the times when we should aim to invoke plugins.